packages/ckeditor5-coremedia-link/__tests__/linktarget/config/LinkDefaultTargetsConfig.test.ts

```diff
@@ -0,0 +1,42 @@
+/* eslint no-null/no-null: off */
+
+import Config from "@ckeditor/ckeditor5-utils/src/config";
+import { computeDefaultLinkTargetForUrl } from "../../../src/linktarget/config/LinkTargetConfig";
+
+jest.mock("@ckeditor/ckeditor5-utils/src/config");
+
+describe("LinkTargetDefaultsConfig", () => {
+  describe("computeDefaultLinkTargetForUrl", () => {
+    // @ts-expect-error - Requires generic type since CKEditor 37.x.
+    let config: Config;
+
+    beforeEach(() => {
+      config = new Config();
+    });
+
+    test.each`
+      url                          | expectedTarget
+      ${"content:123"}             | ${"_embed"}
+      ${"https://www.example.com"} | ${"_blank"}
+      ${"unexpectedFormat"}        | ${undefined}
+    `("[$#] For the url `$url` a target with the value '$expectedTarget' is expected.", ({ url, expectedTarget }) => {
+      config.set("link.defaultTargets", [
```

Comment:
This can be part of the initialization as it does not depend on any input. 
Maybe the config can be initialized once for the whole test (beforeAll instead of beforeEach).